docs/features/runs/attaching-datastores.md

```diff
@@ -14,6 +14,10 @@ You can mount a datastore to a run to make your experiments run faster! By defau
 grid run main.py --my_data_path /datastores/cats/1
 ```
 
+:::note
+  When using datastores the data are requested from S3 and loaded on demand. These data results are then cached to local disk. Because of this you will see slower data
+  retrieval times during the first epoch of a run.
+:::note
```

Comment:
I would rephrase this as "Because of this you will see faster retrieval times after the first epoch of a run"

Reply:
done.

Reply:
This is not true at all. The files can be temporarily cached in memory, but we don't cache files on disk.

Reply:
@oojo12 fyi